torch_xla/csrc/runtime/env_vars.cc

```diff
@@ -7,16 +7,13 @@ namespace env {
 const char* const kEnvNumTpu = "TPU_NUM_DEVICES";
 const char* const kEnvNumGpu = "GPU_NUM_DEVICES";
 const char* const kEnvNumCpu = "CPU_NUM_DEVICES";
-const char* const kEnvLocalWorker = "XRT_LOCAL_WORKER";
 const char* const kEnvTpuConfig = "XRT_TPU_CONFIG";
 const char* const kEnvDeviceMap = "XRT_DEVICE_MAP";
-const char* const kEnvWorkers = "XRT_WORKERS";
+const char* const kEnvWorkers = "RT_WORKERS";
 const char* const kEnvMeshService = "XRT_MESH_SERVICE_ADDRESS";
 const char* const kEnvWorldSize = "XRT_SHARD_WORLD_SIZE";
 const char* const kEnvMpDevice = "XRT_MULTI_PROCESSING_DEVICE";
-const char* const kEnvHostOrdinal = "XRT_HOST_ORDINAL";
-const char* const kEnvShardOrdinal = "XRT_SHARD_ORDINAL";
-const char* const kEnvStartService = "XRT_START_LOCAL_SERVER";
```

Comment:
again I suspect we don't need these XRT_* env var, @will-cromar  to confirm

Reply:
thanks, will track in ticket too

Reply:
Thanks, updated